examples/hello-world/src/integration_tests.rs

```diff
@@ -0,0 +1,152 @@
+#![cfg(test)]
+#![cfg(not(tarpaulin_include))]
+use crate::contract::{execute, instantiate, query};
+use crate::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+use cosmwasm_std::{Addr, Empty};
+use cw_multi_test::{App, AppBuilder, Contract, ContractWrapper, Executor};
+
+fn mock_app() -> App {
+    AppBuilder::new().build()
+}
+
+pub fn contract_hello_world() -> Box<dyn Contract<Empty>> {
+    let contract = ContractWrapper::new(execute, instantiate, query);
+    Box::new(contract)
+}
+
+struct Env {
+    router: App,
+    owner: Addr,
+    hello_world_addr: Addr,
+}
+
+mod mock {
+    use cosmwasm_std::{
+        to_binary, Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult,
+    };
+    use cw_multi_test::{Contract, ContractWrapper};
+
+    use chainlink_terra::msg::QueryMsg as ChainlinkQueryMsg;
+    use chainlink_terra::state::Round;
+
+    use schemars::JsonSchema;
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+    pub struct InstantiateMsg {}
+
+    #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+    pub enum ExecuteMsg {}
```

Comment:
You can reduce the boilerplate/remove some imports by using `()` (void/empty tuple)
```suggestion
    pub type InstantiateMsg = ();
    pub type ExecuteMsg = ();
```

Reply:
oh nice! i didn't know i could do that.